docs/conf/scalar-loki-stack-custom-values.yaml

```diff
@@ -0,0 +1,50 @@
+promtail:
+  config:
+    snippets:
+      scrapeConfigs: |
+        - job_name: scalardl-ledger
```

Comment:
> Also, can you add comments for each configuration?

Thank you for your advice!
I added comments in https://github.com/scalar-labs/helm-charts/pull/87/commits/0a055deb7db736fc137ae9acdfe949f531da1bd6

> Is it OK to hard-code it?

Yokoyama-san suggests a generic solution [here](https://github.com/scalar-labs/helm-charts/pull/87#discussion_r867753020) by which we can safely scrape all related PODs since the POD (created by Scalar's Helm Charts)'s name can be included.

> If it is Scalar DL specific, then we should name it so.

I am sorry I am not really sure the meaning of naming.
Do you mean to create a new custom value?